src/include/distributed/metadata_sync.h

```diff
@@ -30,12 +30,15 @@ extern char * NodeListInsertCommand(List *workerNodeList);
 extern List * ShardListInsertCommand(List *shardIntervalList);
 extern char * NodeDeleteCommand(uint32 nodeId);
 extern char * ColocationIdUpdateCommand(Oid relationId, uint32 colocationId);
+extern char * CreateSchemaDDLCommand(Oid schemaId);
 
 
 #define DELETE_ALL_NODES "TRUNCATE pg_dist_node"
 #define REMOVE_ALL_CLUSTERED_TABLES_COMMAND \
 	"SELECT worker_drop_distributed_table(logicalrelid) FROM pg_dist_partition"
 #define DISABLE_DDL_PROPAGATION "SET citus.enable_ddl_propagation TO 'off'"
+#define WORKER_APPLY_SEQUENCE_COMMAND "SELECT worker_apply_sequence_command ('%s')"
```

Comment:
We shouldn't have quotes in here, and instead use parameters or `quote_` functions.